packages/frontend/src/store/index.ts

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
We should move all the redux files to `packages/frontend/src/store/features/sqlRunner/store/`. 
Avoid confusion and prevent other devs from using this until we are sure about this tech decision.

Reply:
sounds good!

Reply:
done here: https://github.com/lightdash/lightdash/pull/10589/commits/51aab283e02b2f42ce75ef5cc17d18d6f9a1f951